src/components/layout.js

```diff
@@ -76,6 +76,11 @@ export function Hero({ data = {}, subStyle, altStyle, children }) {
             </H2>
           </div>
         )}
+        {data.ctaline && (
```

Comment:
This is support for an entirely new block that didn't exist before for the `hero` component. I opted for this in order to keep your original wording and still shorten the text on the button itself.